modules/geolocation/geolocation.py

```diff
@@ -129,3 +129,40 @@ def calculate_pixel_to_geo_mapping(self):
 
         # Return matrix product of mappedGeoMatrix and sourcePixelMatrixInverse
         return (mappedGeoMatrix.dot(sourcePixelMatrixInverse))
+
+    def map_location_from_pixel(self, transformationMatrix, pixels):
+        """
+        Maps Geographical Location Coordinates in the destination image
+        
+        Parameters
+        -------
+            transformationMatrix : np.array(shape=(3,3))
+            pixels : np.array(shape=(5,2))
+
+        Returns
+        -------
+            np.array(shape=(5,2))
+        """
+
+        # Express all 2D coordinates of pixels as 3D coordinates with z value = 1
+        pixels = np.insert(pixels, 2, 1, axis = 1)
+
+        # Compute Homogeneous Coordinates: Product of Image Pixels and Coordinates
+        homogeneousCoordinates = np.matmul(transformationMatrix,pixels.T).T
+
+        geoCoordinates = np.empty(shape=(0, 2))
+        
+        # Cycle through all homogenized coordinates of pixels
+        for h in homogeneousCoordinates:
+            # Checking if the homogenized value of Z equals 0. If so, we return an empty array.
+            if h[2] == 0:
+                geoCoordinates = np.vstack((geoCoordinates, np.empty(shape = (2))))
```

Comment:
Use `float(‘inf’)` instead of empty array.

Reply:
Done. I've replaced `np.empty(shape=(2))` with `np.full((2), np.inf)`.